tokio/src/net/unix/ucred.rs

```diff
@@ -161,7 +161,7 @@ pub(crate) mod impl_netbsd {
     }
 }
 
-#[cfg(any(target_os = "dragonfly", target_os = "freebsd"))]
+#[cfg(any(target_os = "dragonfly", target_os = "freebsd", target_os = "nto"))]
```

Comment:
Looking at my diff for tokio 1.18.2, I had enabled the impl_netbsd path, not the impl_bsd path.  I presumably did this as it includes the PID whereas getpeereid() does not.  Any argument why the getpeereid() path should be used over this?

Reply:
I don't know what is the correct answer for this target.

Reply:
You are right, using the impl_netbsd is better. I will update the PR